dataland-frontend/src/components/layout/TheFooter.vue

```diff
@@ -0,0 +1,91 @@
+<template>
```

Comment:
There is still the old "TheFooter" component in our code with the exact same name.  We should remove it before mergin this to main.

Reply:
they are in two different folders /src/components/general/TheFooter.vue and /src/components/layout/TheFooter.vue and in this first phase they have to co-exist

Reply:
Correct, old pages are gonna still showing the old footer.

Reply:
Or different question: Why can't the "old" pages use the new Footer too?

Reply:
They could, but I have no instruction of using it everywhere yet since I am only working on one single page for now.

Reply:
From @fsoffel and my perspective, it is ok to use the new footer everywhere.  We could not find a good reason not to during our discussion.
Therefore feel free to use the new footer on all pages.

Unless you see any problems doing so.  Then we can postpone it.  But if it is just replacing the old one with the new one and everything looks fine, it is ok.